phd-tests/framework/src/disk/crucible.rs

```diff
@@ -338,8 +386,11 @@ impl Inner {
     }
 
     fn vcr(&self, disk_id: Uuid) -> VolumeConstructionRequest {
-        let downstairs_addrs =
-            self.downstairs_instances.iter().map(|ds| ds.address).collect();
+        let downstairs_addrs = self
+            .downstairs_instances
+            .iter()
+            .map(|ds| ds.vcr_address())
```

Comment:
Nit: I'm surprised Clippy hasn't suggested
```suggestion
            .map(Downstairs::vcr_address)
```
since it usually prefers point-free style.

Reply:
Fixed in 65f862f.